server/server.js

```diff
@@ -1598,6 +1598,48 @@ let needSetup = false;
             log.debug("auth", "need auth");
         }
 
+        socket.on("updateManual", async (data, callback) => {
```

Comment:
Do we actually need an extra endpoint for this?
Why not use the regular save monitor endpoint?

This seems like code that we should already have